python/lsst/meas/modelfit/__init__.py

```diff
@@ -20,7 +20,21 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 from .version import *
-from .modelfitLib import *
+from .common import *
+from .mixture import *
+from .unitSystem import *
 from .priors import *
-from . import psf     # import these into separate namespaces because they just contain plugins,
-from . import cmodel  # which we don't expect users to need to get to outside of their registries.
```

Comment:
There are many symbols defined in `cmodel` and `psf` that were previously defined in `modelfitLib` and hence lifted to `lsst.meas.modelfit`, and those need to be public.  Putting them in a subpackage with the relatively small amount of pure-Python `psf` and `cmodel` code greatly improved the overall organization of this package, and it made it seem silly to treat that pure-Python code differently here.